util/src/main/scala/org/ensime/util/Show.scala

```diff
@@ -0,0 +1,18 @@
+package org.ensime.util
+
+trait Show[T] {
```

Comment:
Including a whole library for one type class seemed a bit extreme... plus you said something about not relying on scala libs?